examples/99_in_tool_selection/README.md

```diff
@@ -0,0 +1,12 @@
+# In tool selection
+
+This demonstrates how a tool can include a selection stage if activated without
+a usable selection. This is how native Move, Scale and Rotate behaves.
+
+This approach can also be used for tools that only works on objects created
```

Comment:
- [ ] Testing a code comment with a checkbox. Is this something that can work in clarifying what should be resolved before merge? Like JIRA?

Reply:
I don't think checkboxes gives any warnings before closing, but maybe can be used manually before resolving conversations.

Reply:
It's possible to set up checks for GitHub PRs. So we might be able to setup a script that checks for checkboxes. Might even be an existing solution we can use. Worth a look at some point as I expect we'll be using github more going forward.